modules/core/src/main/scala/shop/http/routes/secured/CartRoutes.scala

```diff
@@ -24,7 +26,10 @@ final class CartRoutes[F[_]: Sync](
 
     // Add items to the cart
     case ar @ POST -> Root as user =>
-      ar.req.decode[Cart] { cart =>
+      //JsonDecoder[F].asJsonDecode[Cart](ar.req).flatMap { cart =>
+      import org.http4s.circe._
```

Comment:
If this import is at the top, it seems to be problematic. See https://github.com/http4s/http4s/issues/3033